### PR TITLE
Package Clean Up 

### PR DESCRIPTION
**Pull request by kdorheim**
Improve the documentation and vignette examples.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -2,3 +2,8 @@
 ^\.Rproj\.user$
 ^LICENSE\.md$
 data-raw/
+^\.github/workflows$
+^\./ele12345-sup-0001-tables1\.docx$
```
Comment (bpbond) [inline .Rbuildignore:6]: ?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
+	        role = "cre",
+	        comment = c(ORCID = "0000-0001-9525-4633")), 
+	        person("Kalyn", "Dorheim", 
+	        email = "kalyn.dorheim@pnnl.gov",
+	        role = "ctb", 
+	        comment = c(ORCID = "0000-0001-8093-8397")))
+Description: This package provides a felxible interface to the Simple Ecosystem Model (SEM). SEM was originally 
+  documented in ```Dietze, Michael C., and Jaclyn Hatala Matthes. “A General Ecophysiological Framework for Modelling 
+  the Impact of Pests and Pathogens on Forest Ecosystems.” Ecology Letters 17, no. 11 (November 1, 2014): 
+  1418–26. https://doi.org/10.1111/ele.12345.```
```
Comment (bpbond) [inline DESCRIPTION:16]: Nice. Pretty sure _you_ should be creator though!

**Review by bpbond — APPROVED** [review]
Nice @kdorheim ! Thanks.

There are quite a few spelling mistakes in the comments, not a big deal but might want to review with e.g. RStudio's spellchecker turned on.

**Review by bpbond — APPROVED** [review]

**Comment by bpbond** [comment]
Ready to merge from my pov

File: .Rbuildignore (hunk 1 above)
Comment (kdorheim) [reply .Rbuildignore:6]: That file was removed, will update the Rbuild ignore file

**Comment by kdorheim** [comment]
The build is failing because of the vignettes (curses shaking fist in the air) that should be resolved before this PR is merged.

**Pull request merged**